backend/data/final_data/programsProcessed.json

```diff
@@ -852,5 +864,249 @@
+                    "COMM2222": "Industry Experience Program"
+                },
+                "Integrated First Year Courses": {
+                    "type": "core",
+                    "credits_to_complete": "48",
+                    "COMM1180": "Value Creation",
+                    "COMM1140": "Financial Management",
+                    "COMM1190": "Data, Insights and Decisions",
+                    "COMM1170": "Organisational Resources",
+                    "COMM1150": "Global Business Environments",
+                    "COMM1120": "Collaboration and Innovation in Business",
+                    "COMM1100": "Business Decision Making",
+                    "COMM1110": "Evidence-Based Problem Solving"
+                },
+                "Final Year Synthesis": {
+                    "type": "core",
+                    "COMM3030": "Social Entrepreneurship Practicum",
+                    "TABL3033": "UNSW Tax and Business Advisory Clinic",
+                    "COMM3900": "Capstone Project",
+                    "COMM3090": "Synthesis of Learning Experience",
+                    "COMM2244": "Future of Work Project",
+                    "COMM2233": "Industry Consulting Project",
+                    "COMM3020": "Global Business Practicum",
+                    "ACCT3583": "Management Accounting 2",
+                    "COMM3501": "Quantitative Business Analytics",
+                    "ACTL3192": "Retirement Saving and Spending Over the Lifecycle",
+                    "INFS3020": "International Information Systems and Technology Practicum",
+                    "MGMT3004": "Solving Complex Strategy and Policy Problems",
+                    "ECON3000": "Evidence-based Intervention Design and Evaluation"
+                },
+                "myBCom": {
+                    "type": "core",
+                    "COMM3999": "myBCom Graduation Portfolio",
+                    "COMM0999": "myBCom Blueprint",
+                    "COMM1999": "myBCom First Year Portfolio"
+                }
+            },
+            "Minors": {
+                "MARKA2": "Marketing",
+                "ACCTA2": "Accounting",
+                "FINSA2": "Finance",
+                "MGMTA2": "Management",
+                "MGMTH2": "Human Resource Management",
+                "TABLC2": "Taxation",
+                "COMMJ2": "Business Analytics",
+                "INFSA2": "Information Systems",
+                "ECONF2": "Business Economics",
+                "TABLA2": "Business Law",
+                "FINSR2": "Financial Technology",
+                "IBUSA2": "International Business"
+            }
+        }
+    },
+    "3785": {
+        "title": "Engineering (Honours) / Computer Science",
+        "code": "3785",
+        "duration": 5,
+        "UOC": 240,
+        "faculty": "Faculty of Engineering",
+        "components": {
+            "SpecialisationData": {
+                "Majors": {
+                    " Computer Science": {
```

Comment:
Perhaps use .strip() to get rid of the leading/trailing whitespace on keys like this one

Reply:
whoops nice find ty ty :)